server/monitor_test.go

```diff
@@ -377,6 +377,46 @@ func TestMonitorServerz(t *testing.T) {
 	monitorExpectStatus(t, ServerPath, http.StatusInternalServerError)
 }
 
+func TestMonitorIsFTActiveStandaloneServer(t *testing.T) {
+	expectedStatusCode := http.StatusNoContent
+
+	resetPreviousHTTPConnections()
+	s := runMonitorServer(t, GetDefaultOptions())
+	defer s.Shutdown()
+
+	resp, _ := getBodyEx(t, http.DefaultClient, "http", IsFTActivePath, expectedStatusCode, "")
+	defer resp.Body.Close()
+
+	if expectedStatusCode != resp.StatusCode {
+		t.Fatalf("Expected status code %v, got %v", expectedStatusCode, resp.StatusCode)
+	}
+}
+
+func TestMonitorIsFTActiveFTServer(t *testing.T) {
```

Comment:
If you don't mind, we could replace this test with the one below because then we check both the active and standby server.
```
func TestMonitorIsFTActiveFTServer(t *testing.T) {
	for _, test := range []struct {
		name           string
		checkActive    bool
		expectedStatus int
	}{
		{"active", true, http.StatusOK},
		{"standby", false, http.StatusNoContent},
	} {
		t.Run(test.name, func(t *testing.T) {
			resetPreviousHTTPConnections()

			cleanupFTDatastore(t)
			defer cleanupFTDatastore(t)

			nopts := defaultMonitorOptions
			var aNOpts *natsd.Options
			var sNOpts *natsd.Options

			if test.checkActive {
				aNOpts = &nopts
			} else {
				sNOpts = &nopts
			}

			opts := getTestFTDefaultOptions()
			active := runServerWithOpts(t, opts, aNOpts)
			defer active.Shutdown()

			getFTActiveServer(t, active)

			opts = getTestFTDefaultOptions()
			opts.NATSServerURL = "nats://127.0.0.1:4222"
			standby := runServerWithOpts(t, opts, sNOpts)
			defer standby.Shutdown()

			resp, _ := getBodyEx(t, http.DefaultClient, "http", IsFTActivePath, test.expectedStatus, "")
			defer resp.Body.Close()
		})
	}
}
```

Reply:
hey @kozlovic I went ahead and replaced both tests with the one you described as it's much more concise and validates both codepaths. thanks for the contribution!